ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-firewall/ovn-sdn-migration-feature-migration-validate-egress-firewall-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+validate_egressfirewall_cr () {
+  oc get egressfirewall -n test-migration default -o=jsonpath='{.spec}'>current_config
+  echo "$EXPECT_EGRESS_FIREWALL_SPEC" > expected_config
+  if diff <(jq -S . current_config) <(jq -S . expected_config); then
```

Comment:
this might not be working like you think?
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/31633/rehearse-31633-pull-ci-openshift-cluster-network-operator-master-e2e-network-migration/1567080468996689920/artifacts/e2e-network-migration/ovn-sdn-migration-feature-migration-validate-egress-firewall/build-log.txt

at least, this error doesn't seem right does it?
`parse error: Invalid numeric literal at line 1, column 4
`

Reply:
fixed.